### ISSUE TITLE
Mac touchbar debug not workin

### ISSUE THREAD

**Issue by Alexzanderk** [microsoft/vscode#82222]
VSCode v: September 2019 (version 1.39)
OS: 10.14.6 (18G95) Mojave

Not working start button on debug mode!

**Comment by Alexzanderk** [comment]
in code, it does not work, but if I open shortcuts and press start on touchbar - work fine!

**Comment by Alexzanderk** [comment]
![image](https://user-images.githubusercontent.com/16296369/66553551-011d7a00-eb54-11e9-94f0-b2a2ec1767fa.png)
on this screen - NOT WORK button start

when I open shortcuts 
![image](https://user-images.githubusercontent.com/16296369/66553689-3f1a9e00-eb54-11e9-849a-8486ae3dca34.png)

click on touchbar button start I have this result 

![image](https://user-images.githubusercontent.com/16296369/66553731-5063aa80-eb54-11e9-86ad-5e535de0ba6a.png)

**Comment by isidorn** [comment]
It works for me on MacOS Catalina.
When you say it is not working, does it mean it is not present in the touchbar or clicking on it has no effect?
Does clicking on other debug touchbar buttons has any effect (like once you start debugging can you press the `step` touch bar buttons?)

Also can you reproduce with vscode insiders https://code.visualstudio.com/insiders/

Thanks

**Comment by isidorn** [comment]
@Alexzanderk Thank you for providing details. I can reproduce this.

**Comment by Alexzanderk** [comment]
link on video 
[https://youtu.be/aT3jjAY3FN0](https://youtu.be/aT3jjAY3FN0)

**Comment by Alexzanderk** [comment]
@isidorn In the video, I clicked on `start` button when debugger stop on the mark but nothing was happening and when I open shortcuts  - and click `start` button I continue

**Comment by isidorn** [comment]
Fixed. Will be a part of the recovery release some time next week. Thanks!

**Comment by weinand** [comment]
The "Start" button works fine; the real problem was that the "continue" button (which shares the same location with "start") does not work.

I've verified on Mojave that the touchbar "continue" works fine now.

**Comment by isidorn** [comment]
Exactly. Also the step over / step in were not working. Now they work.

**Comment by Hamlet-Tamaz** [comment]
Now, after starting the debugger, the play buttons disappear from the touchbar, but the other (continue, step-over, etc) don't appear instead. You must click on specific portions of the VSCode  window in order to be in the right environment to have access to the touchbar buttons. This used to work fine  for me until a recent update.

**Comment by isidorn** [comment]
https://github.com/microsoft/vscode/issues/84189
